### PR TITLE
feat(tooltip): swapped to use floatingui

### PR DESCRIPTION
**Pull request by agliga**
## Description

Added floating ui to tooltip. 
At the moment left it to use limited positions. 
There might be a bug with positions but I think its a skin issue. 
This depends on the typescript PR to be merged first.

### CODE REVIEW

**Review by LuLaValva — APPROVED** [review]
Looks good, thanks Andrew! The only problem I noticed was that popper didn't change when the size of the container it was labeling changed (i. e. via screen resize), but I think that may be a limitation of floatingui and not your implementation
image

File: src/components/components/ebay-tooltip-base/constants.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export const pointerStyles:{[index: string]: string} = {
```
Comment (LuLaValva) [inline src/components/components/ebay-tooltip-base/constants.ts:1]: If you want to be more precise with this type, you can say that the value is `Placement`

**Review by LuLaValva — APPROVED** [review]
Quick comment, but you don't have to change it. I think you're good to merge.

File: src/components/components/ebay-tooltip-base/constants.ts (hunk 1 above)
Comment (agliga) [reply src/components/components/ebay-tooltip-base/constants.ts:1]: Good call... Ill swap it.

**Pull request merged**